NOTICE

```diff
@@ -1,4 +1,4 @@
 harbor-scanner-aqua
-Copyright 2019-2020 Aqua Security Software Ltd.
+Copyright 2019-2021 Aqua Security Software Ltd.
```

Comment:
Nice - I wonder if there's an automated way to do this across all our projects.

Reply:
I'm not aware of any functionality like that 😿